lib/widgets/components/entry/register/register_name_text_field.dart

```diff
@@ -29,7 +29,7 @@ class _RegisterNameTextFieldState extends State<RegisterNameTextField> {
     _showError = true;
     await _validateName(name.trim());
     if (!mounted || _error != null) return;
-    //FIXME: warning \/
+    //FIXME:
```

Comment:
Annoying warning that doesn't show up on my machine. (Or in GitHub Actions)
We already have accounted for the warning by checking for `!mounted` in line 31.
Perhaps the only solution would be to split that `if` check into two `if` checks.